### DISCUSSION TITLE
Quarkus Newsletter #22 - July

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#26774] [Quarkus Blog/Website]
# Quarkus Newsletter #22 - July

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-newsletter-22/

**Comment by preslavrachev** [comment]
These are very insightful, but I kind of wished there were a greater variety of topics. In particular, regular old Web development. I know that Quarkus' focus remains on the cloud/container front, but it won't change the fact that it is also a fantastic alternative to Spring and Spring Boot on the Web front. 

That said, I will install the bookmarklet and start submitting the matching stories I find.
